### DISCUSSION TITLE
Traitor Item: CamJam(tm)

### DISCUSSION THREAD

**Discussion by CdrCross** [unitystation/unitystation#8346] [Ideas]
### Description
The CamJam would be a traitor item that takes the form of a handheld remote. When used at range on a security camera, it forces that camera to show a static image (chosen at the the time of the CamJam usage) to the AI and Security Monitoring for the next five minutes, akin to a camera feedback loop. It would have a limited number of charges.

This tool would serve to help traitors prepare for criminal activities by blinding cameras in the area without immediately alerting the AI or security that something is wrong. This would help stealth gameplay for traitors and help to rein in the observational power of the AI a little. The balancing of the CamJam can be adjusted through its TC cost, the number of charges it carries, and how long the camera is affected.

**Comment by PerfectTangent** [comment]
let's keep novel gameplay ideas in the ideas page. That's what the section is for. Converting to discussion.

**Reply by CdrCross** [reply]
Fair enough. Didn't even know this section was used.

**Comment by wraith-54321** [comment]
Camera bugs already do this, but they last forever, let you see through cams, cost 1 TC, and are still only used once in a blue moon.

**Reply by wraith-54321** [reply]
Most people just cut or multitool cams, multitool instead of turning off a cam just makes it only have a 3 tile vision range.
